lib/BrowserHttpFetcher.js

```diff
@@ -2,16 +2,17 @@
 /** A HttpFetcher downloads documents through HTTP. This is the browser implementation */
 
 var q = require('q'),
-    http = require('http'),
-    https = require('https'),
-    util = require('util'),
-    URLParser = require('url'),
-    EventEmitter = require('events');
+  http = require('http'),
+  https = require('https'),
+  util = require('util'),
+  URLParser = require('url'),
+  EventEmitter = require('events');
 
 // Creates a new HttpFetcher - maxParallel is handled by the browser, so we'll ignore it
-function HttpFetcher(accept, maxParallel) {
+function HttpFetcher(accept, customHeaders) {
```

Comment:
we should also have maxParallel en enableCache as properties to have the same interface outside of the browser